### ISSUE TITLE
Question about Loss Function in Chapter 3's Multi-Class Classification

### ISSUE THREAD

**Issue by julian-22py** [StatQuest/signa#11]
In Chapter 3, I noticed the `MultipleInsOuts` class uses `MSELoss` for a multi-class classification problem. I also saw a commented-out line for CrossEntropyLoss in the code.
I'm wondering if there's a reason for using `MSELoss` instead of `CrossEntropyLoss` in this case? Could this be unintentional, or is there a specific advantage to using MSELoss here that I'm not aware of?

**Comment by StatQuest** [comment]
This was intentional. The book doesn't introduce cross entropy until the end of chapter 4, so I don't use it in the coding tutorial for chapter 3.
